packages/x-components/src/components/base-dropdown.vue

```diff
@@ -45,7 +47,7 @@
             :class="itemsCSSClasses[index]"
             class="x-dropdown__item"
             data-test="dropdown-item"
-            role="option"
+            role="menuitem"
```

Comment:
I guess that if we end up changing the top button `role` to `listbox` this should be reverted to `option`

```suggestion
            role="option"
```